packages/analyzer/README.md

```diff
@@ -1,499 +1,3 @@
 # @custom-elements-manifest/analyzer
```

Comment:
I know this goes a little bit against the single source of truth, but can we add the short introduction text and maybe the CLI config table here? Just so that if people see it on NPM, they still get some basic information on how to run it without having to go to a docs site

Reply:
> Looks like there are some styling issues with the code tabs:

Try killing the service worker

> When on the landing page, and you click "analyzer", it shows both "Getting started" and "plugins" open, should only "getting started" be open?

No, each are sub sections in their own right. At least, that's how rocket sees it by default

> Can we add a link to the playground here?

yep